### ISSUE TITLE
Traceback when translating the LICENSE setting in conf.py

### ISSUE THREAD

**Issue by e2jk** [getnikola/nikola#1571]
When setting the `LICENSE` configuration parameter to a simple string (e.g. `LICENSE = "Works fine"`), all works fine. But when setting that parameter to a dict in order to translate it (as this parameter is indicated as being translatable), such as e.g. `LICENSE = {"en": "My Blog", "es": "Mi Blog"}`) the following traceback is thrown:

```
$ nikola build
Traceback (most recent call last):
  File "/usr/local/bin/nikola", line 9, in <module>
    load_entry_point('Nikola==7.2.0', 'console_scripts', 'nikola')()
  File "/usr/local/lib/python2.7/dist-packages/nikola/__main__.py", line 118, in main
    site = Nikola(**config)
  File "/usr/local/lib/python2.7/dist-packages/nikola/nikola.py", line 434, in __init__
    self.config['CONTENT_FOOTER'].langformat(self.config['CONTENT_FOOTER_FORMATS'])
  File "/usr/local/lib/python2.7/dist-packages/nikola/utils.py", line 393, in langformat
    v = TranslatableSetting('NULL', v)
TypeError: __init__() takes exactly 4 arguments (3 given)
```

The form of the `CONTENT_FOOTER` parameter doesn't play a role in this issue, as the 2 forms (string or dict) work fine when the `LICENSE` parameter is a string, and the traceback is thrown in both cases when `LICENSE` is a dict, regardless if `CONTENT_FOOTER` was a string or dict.

Running Nikola v7.2.0 on Ubuntu 14.04, installed via pip.

**Comment by Kwpolska** [comment]
There, fixed in 67c5fb2.  Apparently, the signature of `TranslatatableSetting` was changed in f217ba7, but it was not reflected in `langfrormat`.

**Comment by e2jk** [comment]
Thanks for this fast fix!
